SystemInfo/Value/ComposerPackage.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * File containing the ComposerPackage class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo\Value;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+/**
+ * Value for information about a Composer package.
+ */
+class ComposerPackage extends ValueObject implements SystemInfo
+{
+    /**
+     * Version.
+     *
+     * Example: v2.7.10
+     *
+     * @var string
+     */
+    public $version;
+
+    /**
+     * Date and time string.
+     *
+     * Example: 2016-02-28 20:37:19
+     *
+     * @var string
+     */
+    public $dateTimeString;
```

Comment:
Is it okay to keep this as a string? Or should we convert it, to be more flexible in how we display it?